climpred/classes.py

```diff
@@ -152,6 +153,113 @@ def __repr__(self):
         else:
             return _display_metadata(self)
 
+    def _math(self, other, operator):
+        """Helper function for __add__, __sub__, __mul__, __truediv__.
+
+        Allows math operations with type:
+            - int
+            - float
+            - np.ndarray
+            - xr.DataArray without new dimensions
+            - xr.Dataset without new dimensions or variables
+
+        """
+        assert isinstance(operator, str)
+
+        def add(a, b):
+            return a + b
+
+        def sub(a, b):
+            return a - b
+
+        def mul(a, b):
+            return a * b
+
+        def div(a, b):
+            return a / b
+
+        ALLOWED_TYPES_FOR_MATH_OPERATORS = [
+            int,
+            float,
+            np.ndarray,
+            xr.DataArray,
+            xr.Dataset,
+            type(self),
+        ]
+        # catch other dimensions in other
+        error_str = f'Cannot use {type(self)} {operator} {type(other)}'
```

Comment:
Maybe for aesthetics, add with the other constant:

```python
OPERATOR_STR = {
            'add': '+',
            'sub': '-',
            'mul': '*',
            'div': '/',
        }
```

Then do 
`error_str = f'Cannot use {type(self).__name__} {OPERATOR_STR[operator} {type(other).__name__}'`